### DISCUSSION TITLE
Carelink Follower

### DISCUSSION THREAD

**Discussion by 808IUFan** [NightscoutFoundation/xDrip#3781] [Q&A]
Did anyone else lose their abiltiy to use Carelink Follower a week ago today, 11/15/24, 
 like me? I am updated to the latest version. This is similar to when Medtronic had issues a few months ago but they are working fine now. I will also say that I am not able to even log into the Carelink site on my phone but it works on the web and their app is fine. It seems they have started rejecting all connections outside of their own app but that is why I am posting to see if anyone else is experiencing the same thing. 
<img width="202" alt="Screenshot_20241115-165019" src="https://github.com/user-attachments/assets/b19683b2-b46d-4017-ba42-92d84ef22458">

Update from just today. I can now log into the website on my phone. I have been checking several times a day for that past week and in Chrome on my phone that now allows me to log into Carelink's website.

**Comment by Navid200** [comment]
Post a screenshot of the classic status page.

**Reply by 808IUFan** [reply]
![Screenshot_20241122-134103](https://github.com/user-attachments/assets/67fc341a-5354-4a61-b27f-32bb39822fcb)

**Reply by Navid200** [reply]
According to that screenshot, you are using our latest stable release.
If you look at the [release notes](https://navid200.github.io/xDrip/docs/ReleaseNotes.html), you will see that since then, there have been many Nightly releases.  You will also see that our last Nightly release addresses CareLink.

Would you please install that and try?
The release notes have links to every release.  

In case that is confusing, this is the release you want (or any after that in the future): https://github.com/NightscoutFoundation/xDrip/releases/tag/2024.11.17

**Reply by 808IUFan** [reply]
Yes I did that when it came out earlier in the week and redid it a few minutes ago. I have even just restarted my phone and I still get the same error message.

**Reply by Navid200** [reply]
Then, why did you post that screenshot?
And why wouldn't you post a screenshot of the version you are using now?

**Reply by 808IUFan** [reply]
I dont understand. You asked me for the screenshot. I already had that version and it's not working for me ,

**Reply by Navid200** [reply]
If the screenshot you have posted is what you are running now, it is not surprising that it's not working for you.

Hopefully, someone else can help you as it is quite possible that we are misunderstanding each other.

Good luck.

**Comment by 808IUFan** [comment]
Not sure if this helps or is obvious from the previous screenshot with the error but this is the error log page 
![Screenshot_20241122-143603](https://github.com/user-attachments/assets/541b4699-3112-4917-a9c9-e192aed273bf)

**Comment by 808IUFan** [comment]
I think you are misunderstanding me and that is confusing to me. Today I have only done what you asked me to do. I already had the lastest release and from being in the troubleshooting business for 20+ years professionally for software, hardware, scripting, dataset analysis etc, there seems to be a java issue. Please to don't be upset with me, I am not an amateur and only want things to work. Whatever the fix was in the 11/17 release did not fix the isssue. My phone will not automatically update even though in the software I have chose for it to do that. I have to manually download it and install it and I think I did that on 11/18 after the carelink modification was made. It is not working for me. The issue was also in the release before that and I believe I had the 11/08 version before I updated to the current. Please do not desert me.

**Reply by Navid200** [reply]
I'm not the person who can fix CareLink.
No one who needs to use xDrip with CareLink should use a version older than 2024.11.17.

There are individuals who struggle with updating xDrip because they are not clear that by default, users don't get Nightly releases.  So, we always ask to see a screenshot of the status page as a confirmation that the user is indeed using the latest Nightly.  Other than that, it also confirms that the user is using a release from the official repository.  

It is quite possible that something is wrong again and even the latest Nightly is failing.  But, the individual who can fix this very likely may have the same request, to see that you are using the latest Nightly.  

It is possible that you experienced the problem with the latest Nightly and decided to go back to 2024.09.17.  There is no justification for that action.  There is no way that would work if the latest Nightly is not.  But, the other way around is a possibility.  
That's why it puzzles me why you have gone back to an older version of xDrip be it our latest stable release.  Our latest stable release will be older than out latest Nightly for the next 5 months.

**Reply by 808IUFan** [reply]
Not sure where you got the info that I ever went backwards. I never did that. I never said I had anything other than the current version before I updated on 11/18. I had the one from the first week of the month and when the problem came up I updated to the 11/15 and on Monday the 18th I updated to the 17th. So since the uninstall and reinstall it is working and going forward, when I have an issue and Medtronic isnt, I will just totally uninstall and reinstall because that fixed it. Thanks for your help.

**Reply by Navid200** [reply]
Your screenshot.

**Reply by 808IUFan** [reply]
Nothing is there and the screenshot of the Classic Status page I sent is gone. We dont need to go back and forth with this. It's working now. You seem to think I am stupid and I am not. I am obviously not perfect and neither are you. Take care and again thanks for your help.

**Reply by Navid200** [reply]
These forums are not just for the person who asks for help.  That would be very inefficient considering how short-handed we are.  They are here to help everyone.  

To everyone who may find this thread:
If you are using CarLink follower, you should use an xDrip version of 2024.11.17 or any after that.  
Do NOT use 2024.09.17.

I spend a lot of time in these forums to support diabetics who use xDrip.  It is my intent to help everyone.  When there is a misunderstanding, which is inevitable because we are limited to text, I would appreciate support from the community instead of criticism and personal remarks.

**Reply by 808IUFan** [reply]
Well if I was using 09.17 I didnt notice it because I deliberately went to get the latest update but you should have said you dont have the latest, look again. You said well I am not surprised with the version you have there. That is condescending especially when I knew I looked for the latest. It is also still in my phone download folder. Again thanks.

**Reply by Navid200** [reply]
> but you should have said you dont have the latest  


That is what I posted.  You can still see it: 
<br/>  
  
![Screenshot 2024-11-22 205203](https://github.com/user-attachments/assets/527c11dd-37f2-49ff-b2e6-edc5264ba042)

**Comment by 808IUFan** [comment]
Ok as a last resort I uninstalled the app and reinstalled it. It's working now. I will do this going forward when something happens.
